### ISSUE TITLE
Upgrade to support deno v1.2.0.

### ISSUE THREAD

**Issue by omaralmgerbie** [eveningkid/denodb#77]
### Error

```
Check https://deno.land/x/denodb/mod.ts
error: TS2345 [ERROR]: Argument of type 'string | URL' is not assignable to parameter of type 'string'.
  Type 'URL' is not assignable to type 'string'.
  return new URL(url).pathname
                 ~~~
    at https://deno.land/x/dex/lib/deps/path/win32.ts:911:18

TS2345 [ERROR]: Argument of type 'string | URL' is not assignable to parameter of type 'string'.
  Type 'URL' is not assignable to type 'string'.
  return new URL(url).pathname;
                 ~~~
    at https://deno.land/x/dex/lib/deps/path/posix.ts:433:18

Found 2 errors.
```

**Comment by eveningkid** [comment]
Duplicate of #78. Sadly there was still no update for the dex library. We're still waiting on them...!
